src/OrleansAWSUtils/Storage/SQSStorage.cs

```diff
@@ -0,0 +1,233 @@
+﻿using Amazon.Runtime;
+using Amazon.SQS;
+using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Amazon.SQS.Model;
+using Orleans;
+using SQSMessage = Amazon.SQS.Model.Message;
+
+namespace OrleansAWSUtils.Storage
+{
+    /// <summary>
+    /// Wrapper/Helper class around AWS SQS queue service
+    /// </summary>
+    public class SQSStorage
+    {
+        /// <summary>
+        /// Maximum number of messages allowed by SQS to peak per request
+        /// </summary>
+        public const int MAX_NUMBER_OF_MESSAGE_TO_PEAK = 10;
+        private const string AccessKeyPropertyName = "AccessKey";
+        private const string SecretKeyPropertyName = "SecretKey";
+        private const string ServicePropertyName = "Service";
+        private Logger Logger;
+        private string accessKey;
+        private string secretKey;
+        private string service;
+        private string queueUrl;
+        private AmazonSQSClient sqsClient;
+
+        /// <summary>
+        /// The queue Name
+        /// </summary>
+        public string QueueName { get; private set; }
+
+        /// <summary>
+        /// Default Ctor
+        /// </summary>
+        /// <param name="queueName">The name of the queue</param>
+        /// <param name="connectionString">The connection string</param>
+        /// <param name="deploymentId">The cluster deployment Id</param>
+        public SQSStorage(string queueName, string connectionString, string deploymentId = "")
```

Comment:
Although it is the kind of change that IMHO is pure cosmetic and just a judgement call since there is no real benefit on that (as you can see on SQL connections which are pure strings), I'll make this change not just here but on the DynamoDBStorage as well, since their connection strings are basically the same. I'll do that in another PR before we release the nuget package itself for AWSUtils among other cosmetic issues.